src/applications/facility-locator/components/ControlResultsHolder.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+function ControlResultsHolder({ children, isSmallDesktop }) {
+  // If the screen is smaller than small desktop we just return the children
+  if (!isSmallDesktop) {
+    return children;
```

Comment:
Cannot be enclosed in a div if not small desktop or smaller because grouping and orientation is completely different.